### ISSUE TITLE
[terra-doc-template] Missing themeable variables

### ISSUE THREAD

**Issue by ryanthemanuel** [cerner/terra-core#3049]
# Bug Report

The following themeable variables are missing:

**clinical-lowlight-theme**

--terra-doc-template-doc-card-not-empty-border-radius - packages/terra-doc-template/lib/DocTemplate.module.scss (line: 18, column: 4)
--terra-doc-template-doc-card-not-empty-padding-bottom - packages/terra-doc-template/lib/DocTemplate.module.scss (line: 19, column: 4)
--terra-doc-template-doc-card-not-empty-padding-left - packages/terra-doc-template/lib/DocTemplate.module.scss (line: 20, column: 4)
--terra-doc-template-doc-card-not-empty-padding-right - packages/terra-doc-template/lib/DocTemplate.module.scss (line: 21, column: 4)
--terra-doc-template-doc-card-not-empty-padding-top - packages/terra-doc-template/lib/DocTemplate.module.scss (line: 22, column: 4)

**orion-fusion-theme**

--terra-doc-template-content-background-color - packages/terra-doc-template/lib/ExampleTemplate.module.scss (line: 15, column: 4)
--terra-doc-template-doc-card-not-empty-background-color - packages/terra-doc-template/lib/DocTemplate.module.scss (line: 17, column: 4)
--terra-doc-template-doc-card-not-empty-border-radius - packages/terra-doc-template/lib/DocTemplate.module.scss (line: 18, column: 4)
--terra-doc-template-doc-card-not-empty-padding-bottom - packages/terra-doc-template/lib/DocTemplate.module.scss (line: 19, column: 4)
--terra-doc-template-doc-card-not-empty-padding-left - packages/terra-doc-template/lib/DocTemplate.module.scss (line: 20, column: 4)
--terra-doc-template-doc-card-not-empty-padding-right - packages/terra-doc-template/lib/DocTemplate.module.scss (line: 21, column: 4)
--terra-doc-template-doc-card-not-empty-padding-top - packages/terra-doc-template/lib/DocTemplate.module.scss (line: 22, column: 4)
--terra-doc-template-examples-header-border-bottom - packages/terra-doc-template/lib/DocTemplate.module.scss (line: 3, column: 4)
--terra-doc-template-examples-header-color - packages/terra-doc-template/lib/DocTemplate.module.scss (line: 4, column: 4)
--terra-doc-template-examples-header-text-shadow - packages/terra-doc-template/lib/DocTemplate.module.scss (line: 6, column: 4)
--terra-doc-template-footer-background-color - packages/terra-doc-template/lib/ExampleTemplate.module.scss (line: 24, column: 4)
--terra-doc-template-header-background-color - packages/terra-doc-template/lib/ExampleTemplate.module.scss (line: 9, column: 4)
